docroot/public/js/map.js

```diff
@@ -40,20 +40,72 @@ function renderMap() {
     var items = filteredResults.results;
 
     $.each(items, function (key, val) {
-      var marker = new google.maps.Marker({
-        position: {lat: val.lat, lng: val.lng},
-        type: 'info',
-        icon: 'public/images/person.png',
-        map: map
-      });
-      var html = '<b>' + val.ProviderName + '</b><hr/>' + val.PhysicalCity + ',<br/>' + val.ProviderTypeDescription + '<br />'
-        + val.PhoneNumber + '<br />' + val.QualityRating + ' stars<hr />';
-
-      var infoWindow = new google.maps.InfoWindow;
-      google.maps.event.addListener(marker, 'click', function () {
-        infoWindow.setContent(html);
-        infoWindow.open(map, this);
-      });
+      /**
+       * ProviderType Legend
+       * 1) ?
+       * 2) Licensed Slot Contractor
```

Comment:
I don't think these are the correct provider types.... Please see https://trello.com/c/hnwtGVCc/37-as-a-parent-i-want-better-names-for-the-provider-types-that-mean-something-to-me